Tina-ttrimmis/Homework-3/json-homework/json-hw.js

```diff
@@ -0,0 +1,50 @@
+
+/*
+1. Convert the string `jsonString` to a Javascript Object and store it in a variable
+called `flickrObj`
+*/
+let flickrObj = JSON.parse(jsonString);
+console.log(flickrObj);
+
+
+
+/*
+2. Iterate (this means 'use a loop') and do a console.log of each of the item titles
+with the date the photo was taken. The first console.log should return the following:
+
+"Edelweiss Air, Airbus A320-214, HB-IJU 2016-02-23T18:48:14Z"
+
+*/
+let i;
+for (i=0; i<flickrObj.items.length; i++) {
+	console.log("Title: " + flickrObj.items[i].title);
+	console.log("Date Taken: " + flickrObj.items[i].date_taken);
+}
+
+
+
+/*
+3. Create a new array called 'links'. Iterate through the items array in flickrObj and store the links to each item in the 'links' array.
+*/
+let links = [];
+//let x;
+//for (x=0; x<flickrObj.items.length; x++) {
+	//console.log("Link: " + flickrObj.items[x].link);
+//}
+for(let index = 0;  index < flickrObj.items.length; index++) {
```

Comment:
And here you're declaring and assigning in a single statement, and adding spaces around operators! Nice!